src/Microsoft.DotNet.ProjectModel/OutputPathCalculator.cs

```diff
@@ -10,7 +11,10 @@ public class OutputPathCalculator
     {
         private const string ObjDirectoryName = "obj";
 
-        private readonly ProjectContext _project;
+        private readonly Project _project;
```

Comment:
Why did you replace this? I thought the ProjectContext was exactly a projection of the project for a particular framework/rid. Wrong interpretation?